composer.lock

```diff
@@ -6316,7 +6318,13 @@
                 "type": "zip",
                 "url": "https://api.github.com/repos/allure-framework/allure-codeception/zipball/a69800eeef83007ced9502a3349ff72f5fb6b4e2",
                 "reference": "a69800eeef83007ced9502a3349ff72f5fb6b4e2",
-                "shasum": ""
+                "shasum": "",
+                "mirrors": [
```

Comment:
@vidyli,
oh... we shouldn't have mirrors section in the composer.lock files.
Could you configure your composer to not use mirrors and run the `composer update` command?